src/content/documentation/publish/self-distribution.md

```diff
@@ -61,6 +61,53 @@ When you choose to distribute an add-on yourself, they can be installed using th
   content: content
 %}
 
+{% capture content %}
+
+### Web download
+
+When distributing an add–on from your website, you have two options for making the signed add–on file available for download:
+
+- From a link on a webpage, like this:
+  ```html
+  <div id="example-option-1" class="install-ok">
+    <a href="https://example.com/path/to/extension.xpi">
+      Install my add-on
+    </a>
+  </div>
+  ```
+  This is the recommended approach.
+
+- Using JavaScript, like this:
+  ```html
+  <div id="example-option-2" class="install-ok">
+    <button>
+      Install my add-on
+    </button>
+    <script>
+      document.querySelector("#example-option-2 > button").onclick = () => {
+        window.location = "https://example.com/path/to/extension.xpi";
+      };
+    </script>
+  </div>
+  ```
+  If you take this approach, make sure that the browser detects your JavaScript code as handling user input, for example, it is called from inside a DOM click event.
+  
+When using either method, if the download link or triggering JavaScript is not on the top-level frame of the website, none of the frames between the top-level page and the frame containing the download or JavaScript can be third-party or cross-origin frames.
+
+::: note
+You may find references to using the `InstallTrigger` API to trigger a download from JavaScript. This API was never officially supported and will be removed from Firefox 102. Use is therefore not recommended.
+:::
+
+Using either method, when the user downloads the signed add-on file Firefox installs it.
```

Comment:
There is another "using either method" a few lines before (at line 95), not a big deal but maybe we could use a slightly different wording in one of them.

Reply:
Good point @rpl as in the first instance we mention both methods in the sentence, I've removed this introductory phrase from there as it seems somewhat unnecessary.